contrib/testgen/gen_key_io_test_vectors.py

```diff
@@ -65,29 +67,39 @@
 ]
 # templates for valid bech32 sequences
 bech32_templates = [
-  # hrp, version, witprog_size, metadata, output_prefix
-  ('bc',    0, 20, (False, 'main',    None, True), p2wpkh_prefix),
-  ('bc',    0, 32, (False, 'main',    None, True), p2wsh_prefix),
-  ('bc',    1,  2, (False, 'main',    None, True), (OP_1, 2)),
```

Comment:
In commit "Use Bech32m encoding for v1+ segwit addresses" (ea3995aa3cd8f04d7328c95a43c79a5b20fa18a5)

I probably just need to stare at this more, but maybe you could summarize the changes here. Like where did the 1,2 and 2,16 cases go?

Reply:
They're just gone and replaced by better cases. I got rid of (1,2) as (1,32) is a more relevant case (that's taproot) and replaced it with (2,2) instead. To test a bit larger variety, some other (2, _) were replaced with (3, _) too.